src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -178,7 +182,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             DockerAccess access = null;
             try {
                 access = createDockerAccess(minimalApiVersion);
-                ServiceHub serviceHub = serviceHubFactory.createServiceHub(project, session, access, log, logSpecFactory);
+                fetchLimit = resolvedImages.get(0).getRunConfiguration().getFetchLimit() != null
```

Comment:
Just saw that there are new filters for the list call:
````
exited=<int>; -- containers with exit code of <int> ;
status=(created|restarting|running|paused|exited|dead)
label=key or label="key=value" of a container label
isolation=(default|process|hyperv) (Windows daemon only)
ancestor=(<image-name>[:<tag>], <image id> or <image@digest>)
before=(<container id> or <container name>)
since=(<container id> or <container name>)
volume=(<volume name> or <mount point destination>)
network=(<network id> or <network name>)
````

So maybe `since` and `ancestors` could be a match for our use case ....